config/initializers/action_cable_patch.rb

```diff
@@ -0,0 +1,45 @@
+ActiveSupport.on_load(:action_cable) do
+  require 'action_cable/subscription_adapter/postgresql'
+  ActionCable::SubscriptionAdapter::PostgreSQL.class_eval do
+    def new_connection
```

Comment:
The ultimate fix is probably going to look a bit different. Safest would probably be to trap on a version bump, and review at that time (moderate chance the fix won't make 5.0.1, so the required action may be to just increment the check)